Added OpenCms and CKAN

seralf opened the pull request:
Added OpenCms support. This can be improved.

seralf commented:
Hi what about these addictions? if there are still things to fix I can fix them, please let me know

ElbertF commented on share/apps.json:
```diff
@@ -620,6 +620,15 @@
 			"env": "ch_c(?:lient|olor_site_link)"
 		},
+		"Ckan": {
+			"website": "http://ckan.org/",
```
Please remove the protocol from the URL, i.e. `ckan.org`.

seralf replied:
ok fixed!

ElbertF replied:
It's not fixed yet. :)

seralf replied:
wow this is strange! I'll check why the push doesn't work, sorry

2015-01-26 23:43 GMT+01:00 Elbert Alias <notifications@github.com>:

> In share/apps.json
> <https://github.com/ElbertF/Wappalyzer/pull/753#discussion_r23571426>:
>
> > @@ -620,6 +620,15 @@
> >  			"script": "scripts\\.chitika\\.net/",
> >  			"env": "ch_c(?:lient|olor_site_link)"
> >  		},
> > +		"Ckan": {
> > +			"website": "http://ckan.org/",
>
> It's not fixed yet. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ElbertF/Wappalyzer/pull/753/files#r23571426>.
>

seralf replied:
ah ok, I modified the Link, not the url... :-)


2015-01-26 23:44 GMT+01:00 Alfredo Serafini <seralf@gmail.com>:

> wow this is strange! I'll check why the push doesn't work, sorry
>
> 2015-01-26 23:43 GMT+01:00 Elbert Alias <notifications@github.com>:
>
>> In share/apps.json
>> <https://github.com/ElbertF/Wappalyzer/pull/753#discussion_r23571426>:
>>
>> > @@ -620,6 +620,15 @@
>> >  			"script": "scripts\\.chitika\\.net/",
>> >  			"env": "ch_c(?:lient|olor_site_link)"
>> >  		},
>> > +		"Ckan": {
>> > +			"website": "http://ckan.org/",
>>
>> It's not fixed yet. :)
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/ElbertF/Wappalyzer/pull/753/files#r23571426>.
>>
>
>

ElbertF commented on share/apps.json:
```diff
@@ -620,6 +620,15 @@
+				"Access-Control-Allow-Headers": "X-CKAN-API-KEY",
+                "Link": "<http://ckan.org/>; rel=shortlink" 
+            },
+            "implies": [ "python", "solr", "java", "postgres\\;confidence:80" ]
```
Please match the case of the implies applications, i.e. `Python` instead of `python`.

seralf commented:
I have fixed both the things, and added references to Solr/Lucene as libraries used. At the moment I added them under the category "databases", maybe a search frameworks or a similar category could be added later, to be more precise.

ElbertF commented on share/apps.json:
```diff
@@ -620,6 +620,15 @@
+			"cats": [ 1 ],
+			"headers": { 
+				"Access-Control-Allow-Headers": "X-CKAN-API-KEY",
+                "Link": "<ckan.org/>; rel=shortlink" 
```
Will this still match now that you changed the link? I'm guessing it won't.

seralf replied:
ahh ok, I have to mantai the scheme here, sorry :-D

2015-01-26 23:47 GMT+01:00 Elbert Alias <notifications@github.com>:

> In share/apps.json
> <https://github.com/ElbertF/Wappalyzer/pull/753#discussion_r23571703>:
>
> > @@ -620,6 +620,15 @@
> >  			"script": "scripts\\.chitika\\.net/",
> >  			"env": "ch_c(?:lient|olor_site_link)"
> >  		},
> > +		"Ckan": {
> > +			"website": "ckan.org/",
> > +			"cats": [ 1 ],
> > +			"headers": {
> > +				"Access-Control-Allow-Headers": "X-CKAN-API-KEY",
> > +                "Link": "<ckan.org/>; rel=shortlink"
>
> Will this still match now that you changed the link? I'm guessing it won't.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ElbertF/Wappalyzer/pull/753/files#r23571703>.
>

seralf commented:
ok, from errors to typos to errors again, It should be ok now :-)

ElbertF commented:
Cheers!

seralf commented:
:-)
